src/uglify/index.js

```diff
@@ -1,10 +1,9 @@
 import os from 'os';
+import cacache from 'cacache';
```

Comment:
`cacache` => `cache`

Reply:
I would stay with cacache as name. name is usual equal to package name.